### PR TITLE
removed flex-shrink from the room img

### PR DESCRIPTION
**Pull request by kakarda653**
Fixes sparkletown/internal-sparkle-issues#721, when mousing over certain rooms, their aspect ratio changes. This only seems to happen with some images - but not others

### CODE REVIEW

**Review by mike-lvov — APPROVED** [review]
LGTM, tested in https://6.sparkle.space/in/floatycitydemo

**Comment by 0xdevalias** [comment]
@mike-lvov This seems to be stuck on CI linting, I haven't looked into why:

image

File: src/components/templates/PartyMap/components/Map/MapRoom.scss [hunk 1]
```diff
@@ -60,6 +60,7 @@ $label-visible: flex;
 
   &__image {
     position: relative;
+    flex-shrink: 0;
```
Comment (0xdevalias) [inline src/components/templates/PartyMap/components/Map/MapRoom.scss:63]:
```suggestion
    flex-shrink: 0;
```

File: src/components/templates/PartyMap/components/Map/MapRoom.scss [hunk 2]
```diff
@@ -60,6 +60,7 @@ $label-visible: flex;
 
   &__image {
     position: relative;
+    flex-shrink: 0;FOO
```
Comment (0xdevalias) [inline src/components/templates/PartyMap/components/Map/MapRoom.scss:63]:
```suggestion
    flex-shrink: 0;
```

**Comment by 0xdevalias** [comment]
RE: https://github.com/sparkletown/sparkle/pull/1438#issuecomment-847627992

@mike-lvov Seems to be working now:

image

**Pull request merged**